server/internal/provider/kaetzchen/cbor_plugins.go

```diff
@@ -162,6 +164,13 @@ func (k *CBORPluginWorker) sendworker(pluginClient *cborplugin.Client) {
 						k.log.Debugf("%v: Failed to generate SURB-Reply: %v (%v)", pluginCap, r.ID, err)
 						continue
 					}
+					// Set the packet queue delay
```

Comment:
doesn't seem fix the delay problem, you are applying the delay after the service processes the packet. we need the delay to happen before the service processes the payload.